db/changes_view.go

```diff
@@ -174,6 +174,66 @@ func (dbc *DatabaseContext) getChangesInChannelFromQuery(
 	return entries, nil
 }
 
+// Queries the 'channels' view to get changes from a channel for the specified sequence.  Used for skipped sequence check
+// before abandoning.
+func (dbc *DatabaseContext) getChangesForSequences(sequences []uint64) (LogEntries, error) {
+	if dbc.Bucket == nil {
+		return nil, errors.New("No bucket available for sequence query")
+	}
+	start := time.Now()
+	usingViews := dbc.Options.UseViews
+
+	entries := make(LogEntries, 0)
+
+	base.Infof(base.KeyCache, "Querying for sequences (#keys:%d)", len(sequences))
```

Comment:
I propose we make this log debug, and the `Issuing skipped sequence clean query` log info, as it's further up the stack, and contains more information.

```suggestion
	base.Debugf(base.KeyCache, "Querying for sequences (#keys:%d)", len(sequences))
```